java/compaction/compaction-core/src/main/java/sleeper/compaction/job/CompactionJob.java

```diff
@@ -183,6 +182,7 @@ public Builder iteratorConfig(String iteratorConfig) {
         }
 
         public CompactionJob build() {
+            checkDuplicates(inputFiles);
```

Comment:
The null check needs to happen before this.

Reply:
Thanks, moved the null check to before this call